app/controllers/authenticate_controller.rb

```diff
@@ -8,6 +8,8 @@ def authenticate_okta
     # TODO: need a mechanism for an authenticator strategy to define the required
     # params. This will likely need to be done via the Handler.
     params.permit!
+    raise Errors::Authentication::RequestBody::MissingRequestParam, params[:code] unless params[:code]
+    raise Errors::Authentication::RequestBody::MissingRequestParam, params[:state] unless params[:state]
```

Comment:
Let's move these to the Handler.  This method will eventually be named `authenticate` and be the generic endpoint for GET-based authentication requests.